pkg/helper-job/deploy/change.go

```diff
@@ -100,7 +112,7 @@ func isDeployLiteAPIServer(kubeClient *kubernetes.Clientset, nodeName string) (b
 	}
 
 	if !isRunning && isLabel {
-		if err := deleteLiteAPiServerLabel(kubeClient, nodeName); err != nil {
+		if err := deleteEdgedNodeLabel(kubeClient, nodeName); err != nil {
```

Comment:
alway return false，whether deleteEdgedNodeLabel() return err or not

Reply:
although always return false, but changeNodeJob() function will determine whether it needs to return err based on whether err is nil.